actioncable/lib/action_cable/server/broadcasting.rb

```diff
@@ -20,7 +20,10 @@ module Server
     module Broadcasting
       # Broadcast a hash directly to a named <tt>broadcasting</tt>. This will later be JSON encoded.
       def broadcast(broadcasting, message, coder: ActiveSupport::JSON)
-        broadcaster_for(broadcasting, coder: coder).broadcast(message)
+        payload = { broadcasting: broadcasting, message: message, coder: coder }
+        ActiveSupport::Notifications.instrument("broadcast.action_cable", payload) do
```

Comment:
Where it currently is we would miss notifications if we stored a reference to a broadcaster obtained from `broadcaster_for`. Putting the the notification hook in the Broadcaster class ensures we will notifications on all broadcasts. Thanks for pointing that out!